ddtrace/span.py

```diff
@@ -204,6 +207,10 @@ def set_tag(self, key, value=None):
         elif key == MANUAL_DROP_KEY:
             self.context.sampling_priority = priority.USER_REJECT
             return
+        elif key == SERVICE_VERSION_KEY:
+            # Also set the `version` tag to the same value
```

Comment:
Maybe I'm missing context on what the service version is supposed to be.. why do we want to set the version tag to be the same as service version here?

Reply:
`service.version` is an OpenTelemetry tag, we want to make sure that if someone manually sets `service.version` (or they are using an OpenTelemetry library) that we'll pick up on it.